packages/name-controller/src/NameController.ts

```diff
@@ -14,6 +14,7 @@ import type {
 import { NameType } from './types';
 
 export const FALLBACK_VARIATION = '*';
+export const PROPOSED_NAME_EXPIRE_DURATION = 60 * 60 * 24; // 24 hours
```

Comment:
Ultra minor, but should we have a line break to differentiate the public and internal constants?

Reply:
Fixed in https://github.com/MetaMask/core/pull/3748/commits/f6cf1522aa3aba75d703f2926d120c189c9faab9